### DISCUSSION TITLE
官方文档的组件/API支持情况的编写标准是什么，有些疑惑

### DISCUSSION THREAD

**Discussion by Symous** [NervJS/taro#11569] [Q & A] [answered]
### 相关平台
微信小程序

**小程序基础库: 2.17.0**
**使用框架: React**

### 复现步骤
在查看官方文档中的组件和API的平台支持情况时，发现了两种写法。

一种是把所有的平台都列了一遍，理解为显示的就支持，不显示的就不支持？
![image](https://user-images.githubusercontent.com/13534034/161373739-faca2ef0-3ad8-41f2-9045-d2998f20641f.png)

另外一种只显示了微信小程序、H5、RN等平台少数图标，颜色有彩色和灰色，理解为彩色的支持，灰色的不支持？
![image](https://user-images.githubusercontent.com/13534034/161373777-81e2d4f1-1ba5-4e1e-bf91-6b955694ef13.png)

请问这两种支持情况描述到底有何不同，谢谢。

### 期望结果
...

### 实际结果
...

### 环境信息
```
👽 Taro v3.4.4


  Taro CLI 3.4.4 environment info:
    System:
      OS: macOS 11.6
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 15.4.0 - /usr/local/bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 7.0.15 - /usr/local/bin/npm
    npmPackages:
      @tarojs/cli: 3.4.4 => 3.4.4
      @tarojs/components: 3.4.4 => 3.4.4
      @tarojs/mini-runner: 3.4.4 => 3.4.4
      @tarojs/plugin-framework-react: 3.4.4 => 3.4.4
      @tarojs/react: 3.4.4 => 3.4.4
      @tarojs/rn-runner: 3.4.4 => 3.4.4
      @tarojs/runtime: 3.4.4 => 3.4.4
      @tarojs/taro: 3.4.4 => 3.4.4
      @tarojs/taro-rn: 3.4.4 => 3.4.4
      @tarojs/webpack-runner: 3.4.4 => 3.4.4
      babel-preset-taro: 3.4.4 => 3.4.4
      eslint-config-taro: 3.4.4 => 3.4.4
      react: ^17.0.2 => 17.0.2
      react-native: ^0.67.3 => 0.67.4
    npmGlobalPackages:
      typescript: 3.4.5
```

**Comment by ZakaryCode** [comment, marked as answer]
不显示表示没有写，灰色才是不支持

**Comment by Symous** [comment]
@ZakaryCode 抱歉，还是没太理解，没有写和不支持有什么区别？

**Reply by ZakaryCode** [reply]
> @ZakaryCode 抱歉，还是没太理解，没有写和不支持有什么区别？

没写是说文档没有明确标注（此时以对应平台的官方文档为准），毕竟没有平台都有大量的组件和 API，想要持续全平台同步是很难做到的。不支持是我们特别标注过，当前平台不支持的组件或 API

**Reply by Symous** [reply]
感谢回复，但是我还是很疑惑，以最基础的button组件为例，在官方文档中，仅标注了微信小程序/H5/RN三种平台，难道意思是其他的，比如支付宝等小程序都【没有写】Button组件？
![image](https://user-images.githubusercontent.com/13534034/162419823-2de37836-aaeb-4fb2-b90a-d20f3a0aecc7.png)

但是另一方面，我发现ScrollView组件，标注的就比Button组件的平台多。
![image](https://user-images.githubusercontent.com/13534034/162420551-4253ce34-2a1c-4e57-99d5-8723dc3cb4ff.png)

但明显Button是一个比ScrollView更加基础的组件。因为涉及到的组件/平台比较多，感觉官方文档应该是我们判断支持性的依据，所以目前阅读你们的文档编写规范还是没太消化思路，所以再提问一下，感谢理解。

**Reply by Symous** [reply]
我是不是可以理解为，目前的官方文档平台支持情况这边的标注，贵团队编写和统计的还不是很规范和全面。

**Reply by ZakaryCode** [reply]
> 我是不是可以理解为，目前的官方文档平台支持情况这边的标注，贵团队编写和统计的还不是很规范和全面。

组件和 API 相关的文档是基于代码自动生成的，并没有官方统计，都是社区贡献的相关内容

**Comment by xiaoxiyao** [comment]
确实让人费解，希望能在文档中补充说明
